controller/controller.go

```diff
@@ -27,12 +30,15 @@ import (
 )
 
 const (
+	concurrency = 16
```

Comment:
This should be configurable, but I'm not certain how useful that is. Either it's lower, and users have less concurrency, or higher and users have more unused goroutines.

My hunch is to set it to a default value to begin with. I have some ideas for creating and deleting workers dynamically, so we could eventually get rid of this.